redpanda/examples/post-share-price-schema.sh

```diff
@@ -5,7 +5,9 @@
 # redpanda schema service does not support newlines, and needs the schema embedded in a json object as a
 # value for a "schema" key, which means the keys and values in the schema itself need to have their
 # quotes escaped... what a royal pain.
-PAYLOAD=$(echo -n '{ "schema" : "'; cat avro/share_price.json | sed 's/"/\\"/g' | tr '\n' ' '; echo -n '" }')
+PAYLOAD=$(echo '{ "schema" : "\c'; cat avro/share_price.json | sed 's/"/\\"/g' | tr '\n' ' '; echo '" }\c')
```

Comment:
It doesn't work to change something to work on mac if it ends up broken in linux...

```
cfs@erke 20:58:58 ~/dh/oss1/deephaven-core/redpanda/examples
$ PAYLOAD=$(echo '{ "schema" : "\c'; cat avro/share_price.json | sed 's/"/\\"/g' | tr '\n' ' '; echo '" }\c')

cfs@erke 20:59:07 ~/dh/oss1/deephaven-core/redpanda/examples
$ echo $PAYLOAD 
{ "schema" : "\c { \"type\" : \"record\", \"namespace\" : \"io.deephaven.examples\", \"name\" : \"share_price\", \"fields\" : [ { \"name\" : \"Symbol\", \"type\" : \"string\" }, { \"name\" : \"Side\", \"type\" : \"string\" }, { \"name\" : \"Qty\", \"type\" : \"int\" }, { \"name\" : \"Price\", \"type\" : \"double\" } ] } " }\c
```

Sounds like in linux it needs `-e`, do you have `-e` on mac? (I assume you don't have -n this the change?)
```
cfs@erke 21:00:16 ~/dh/oss1/deephaven-core/redpanda/examples
$ PAYLOAD=$(echo -e '{ "schema" : "\c'; cat avro/share_price.json | sed 's/"/\\"/g' | tr '\n' ' '; echo -e '" }\c')

cfs@erke 21:00:25 ~/dh/oss1/deephaven-core/redpanda/examples
$ echo $PAYLOAD 
{ "schema" : "{ \"type\" : \"record\", \"namespace\" : \"io.deephaven.examples\", \"name\" : \"share_price\", \"fields\" : [ { \"name\" : \"Symbol\", \"type\" : \"string\" }, { \"name\" : \"Side\", \"type\" : \"string\" }, { \"name\" : \"Qty\", \"type\" : \"int\" }, { \"name\" : \"Price\", \"type\" : \"double\" } ] } " }
```

Reply:
echo -n didn't work for me in MacOS, but the ending '\c' should work for both Linux and MacOS.